client/components/gravatar/index.jsx

```diff
@@ -1,37 +1,46 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import url from 'url';
 import qs from 'querystring';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import safeImageURL from 'lib/safe-image-url';
+import {
+	getUserTempGravatar
+} from 'state/current-user/gravatar-status/selectors';
 
-module.exports = React.createClass( {
-	displayName: 'Gravatar',
+export class Gravatar extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
```

Comment:
You mean property initializers right, since we don't want this to be static. 😄  I'm fine with the above usage